pypeit/specobjs.py

```diff
@@ -45,6 +45,9 @@ class SpecObjs:
     """
     version = '1.0.0'
 
+    #TODO JFH This method only populates some of the underlying specobj attributes, for example RA and DEC are not
```

Comment:
are you sure?

I don't see this as possible.

Reply:
Isn't this intentional, RA and DEC are populated for DEIMOS by the code that figures it out based on the mask definitions etc. But that code doesn't exist for other instruments. Or am I completely misunderstanding?